js/mimi.js

```diff
@@ -82,27 +89,33 @@
 
 				var previousNotifications = $wrapper.find( '.mimi-error, .mimi-info' );
 
-				if ( 0 != previousNotifications.length )
+				if ( 0 != previousNotifications.length ) {
 					previousNotifications.remove();
+				}
 
 				$wrapper.prepend( MadMimi.addMessage( 'error', MadMimi.fix ) );
+
 			}
 
 		} );
 	};
 
 	MadMimi.addMessage = function( type, message ) {
+
 		var _class = [];
 
 		if ( $.isArray( type ) ) {
+
 			$.each( type, function( index, value ) {
 				_class.push( 'mimi-' + value );
 			} );
+
 		} else {
 			_class.push( 'mimi-' + type.toString() );
 		}
 
 		return $( '<p/>', { class: _class.join( ' ' ) } ).text( message );
```

Comment:
Nice - this is the best way to create elements in JS and avoid XSS vulnerabilities